Readme.md

```
@@ -44,3 +38,53 @@ LinkedRecords is configured via environment variables. See table below.
 | ENABLE_AUTH_RULE_CACHE | false | Enable cache for authorization lookups. Might require a lot of memory. |
 | SHORT_LIVED_ACCESS_TOKEN_SIGNING | xxxx | Configuring this is optional but can reduce load on the database because short lived access token will be used for checking access when a client subscribes to attribute changes. |
 
+## Confidential Client Mode (Cookies possible)
+
+| FRONTEND_BASE_URL | http://localhost:3001 | The base URL of the frontend. It will be used for the Access-Control-Allow-Origin HTTP header and is also required for the OpenID connect redirection. |
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)